tests/Console/CommandTestCase.php

```diff
@@ -91,8 +94,7 @@ protected function getDisplay(): string
     /**
      * Gets the status code returned by the last execution of the command.
      *
-     * @return int
-     *   The status code.
+     * @return int The exit code.
```

Comment:
Why did this change?

Reply:
Well, I realized that I do the "comment on the next line" bit because it's a part of Drupal's coding standards, but that every other project I work with (e.g., Symfony) does it on the same line. So I decided to switch. As before, let's talk standards at scrum.